R/methods.R

```diff
@@ -1,17 +1,85 @@
-#' Title
+#' Imputation & analysis Methods
+#'
+#' @description
+#'
+#' These functions determine what methods rbmi should use when creating
+#' the imputation models, generating imputed values and  pooling the results.
```

Comment:
and  pooling -> and pooling (remove one blank space)

Reply:
done